modules/porous_flow/doc/content/modules/porous_flow/multiapp_fracture_flow_PorousFlow_3D.md

```diff
@@ -0,0 +1,333 @@
+
+!listing 3dFracture/fracture_only_aperture_changing.i block=PorousFlowFullySaturated
+
+!alert note
+[Kuzmin-Turek](/porous_flow/kt.md) stabilization cannot typically be used for the fracture flow.  This is because multiple fracture elements will meet along a single finite-element edge when fracture planes intersect, while for performance reasons, libMesh assumes only one or two elements share an edge.  This prevents the KT approach from quantifying flows in all neighboring elements, which prevents the scheme from working.
+
+Because large changes of temperature and pressure are experienced in this model, the Water97 equation of state is employed:
+
+!listing 3dFracture/fracture_only_aperture_changing.i block=FluidProperties
+
+
+### Material properties
+
+The insitu fracture aperture is assumed to be $a_{0} = 0.1\,$mm for all the fracture planes.  The fractures are assumed to dilate due to increasing porepressure:
+
+\begin{equation}
+\label{eqn.frac.open}
+a = a_{0} + A(P - P_{0}) \ ,
+\end{equation}
+
+where $A = 10^{-3}\,$m.MPa$^{-1}$.  [eqn.frac.open] could easily be generalised to include a temperature-dependent term (which would still be modelled using [PorousFlowPorosityLinear](PorousFlowPorosityLinear.md)), or more complicated physics introduced through the use of other [PorousFlow porosity](porous_flow/porosity.md) classes.  Using $A = 10^{-3}\,$m.MPa$^{-1}$ means that a pressure increase of 1$\,$MPa dilates the fracture by 1$\,$mm.
+
+The page on [mathematics and physical interpretation](multiapp_fracture_flow_equations.md) demonstrated that the porosity in the 2D fracture simulation should be multiplied by $a$.  Assuming that the porosity of the fracture is 1, the simulation's porosity is $\phi = a$:
+
+!listing 3dFracture/fracture_only_aperture_changing.i block=porosity
+
+The insitu fracture permeability is assumed to be $10^{-11}\,$m$^{2}$.  It is assumed that this is $r a_{0}^{2}/12$, where $r=0.012$ is a factor capturing the roughness of the fracture surfaces.  It is also assumed that when the fracture dilates, this becomes $ra^{2}/12$.  Because of the multplication by $a$ (see [here](multiapp_fracture_flow_equations.md)), the value of permeability used in the simulations is $ra^{3}/12$:
+
+\begin{equation}
+\label{eqn.frac.perm}
+k = ra^{3}/12 = \frac{ra_{0}^{3}}{12} \frac{a^{3}}{a_{0}^{3}} = 10^{-15} \left(\frac{\phi}{\phi_{0}}\right)^{3} \ .
+\end{equation}
+
+Hence a [PorousFlowPermeabilityKozenyCarman](PorousFlowPermeabilityKozenyCarman.md) Material can be used:
+
+!listing 3dFracture/fracture_only_aperture_changing.i block=permeability
+
+Because there is essentially no rock material within the fracture, the matrix internal energy is zero, and it is assumed that the thermal conductivity is constant and independent of $a$ (in reality, this too should increase with $a$, but PorousFlow currently lacks this capability, and thermal conductivity is unlikely to be important within the fracture):
```

Comment:
should it be fracture internal energy is zero?

Reply:
I've expanded this to read:

PorousFlow's "matrix internal energy" quantifies the internal energy of the rock material.  Since there is essentially no rock material within the fracture, the matrix internal energy for the fracture App is zero.  ...